app/controllers/user/dashboard_controller.rb

```diff
@@ -4,7 +4,7 @@ class User::DashboardController < User::BaseController
   # GET
   def index
     @organisations = current_user.organisations
-    @requests = current_user.requests.unresolved
+    @requests = policy_scope(current_user.requests.unresolved, policy_scope_class: RequestPolicy::Scope)
     @documents = current_user.documents
```

Comment:
⚠️ We'll need to put some thought into document scope once we have multiple types of documents